Source/WebKit/Platform/IPC/MessageFlags.h

```diff
@@ -32,6 +32,7 @@ enum class MessageFlags : uint8_t {
     DispatchMessageWhenWaitingForUnboundedSyncReply = 1 << 1,
     UseFullySynchronousModeForTesting = 1 << 2,
     MaintainOrderingWithAsyncMessages = 1 << 3,
+    MessageTestingError = 1 << 4,
```

Comment:
I'd call this something more like SyncMessageDeserializationFailure, and surround it with #if ENABLE(IPC_TESTING_API)